00XX-common-ledger-protocol/00XX-common-ledger-protocol.md

```diff
@@ -0,0 +1,273 @@
+# Common Ledger Protocol (CLP)
+
+## Preface
+
+This document describes the Common Ledger Protocol (CLP), a ledger protocol for
+bilateral liquidity relationships. It is similar in function to [Plugin RPC],
+but has been rewritten to use OER instead of JSON, and extensibility features
+have been added.
+
+## Introduction
+
+### Motivation
+
+There are many different types of ledgers, and Interledger aims to interoperate
+all of them. For fast ledgers that support hashlocks, it is enough to wrap the
```

Comment:
s/hashlocks/hash time locks, memos, and messaging.

You need the timeout for expiresAt, the memo for the ILP packet, and the messaging for ILQP.

They also need to support losslessly transferring an integer amount from any account to any other account and checking your integer balance but we can say if they don't support those then the thing is not a ledger.